src/oge/data_cleaning.py

```diff
@@ -416,12 +416,19 @@ def create_primary_fuel_table(
 
     # Drop columns from manual table in-place before concatenation
     primary_fuel_manual.drop(columns=["year", "notes"], inplace=True)
-    primary_fuel_table = pd.concat(
-        [primary_fuel_table, primary_fuel_manual],
-        axis=0,
-        ignore_index=True,
-        copy=False,
-    ).sort_values(by=["plant_id_eia", "subplant_id"])
+
+    # Only concatenate if primary_fuel_manual is not empty
+    if not primary_fuel_manual.empty:
+        primary_fuel_table = pd.concat(
+            [primary_fuel_table, primary_fuel_manual],
+            axis=0,
+            ignore_index=True,
+            copy=False,
+        ).sort_values(by=["plant_id_eia", "subplant_id"])
+    else:
+        primary_fuel_table = primary_fuel_table.sort_values(
+            by=["plant_id_eia", "subplant_id"]
+        )
```

Comment:
Fixes FutureWarning: The behavior of DataFrame concatenation with empty or all-NA entries is deprecated. In a future version, this will no longer exclude empty or all-NA columns when determining the result dtypes. To retain the old behavior, exclude the relevant entries before the concat operation.